lib/internal/url.js

```diff
@@ -874,32 +871,32 @@ class URLSearchParams {
     this[context] = null;
   }
 
-  [util.inspect.custom](recurseTimes, ctx) {
+  [util.inspect.custom](depth, opts) {
```

Comment:
I prefer the `recurseTimes`/`ctx` wording since that's what's used in [`util.js`](https://github.com/nodejs/node/blob/b2803637e867e47318936f388ba2ec52932111f6/lib/util.js#L356).